.travis.yml

```diff
@@ -0,0 +1,37 @@
+sudo: required
+
+language: ruby
+
+services:
+  - docker
+
+env:
+  - DOCKER_COMPOSE_VERSION=1.14.0
+
+install:
+  # Installing docker
+  - curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
+  - sudo apt-get update
+  - sudo apt-get -y install docker-ce
+
+  # Installing docker-compose
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L "https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-$(uname -s)-$(uname -m)" > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+
+before_script:
+  # Check docker & docker-compose versions
+  - docker --version
+  - docker-compose --version
+
+script:
+  - docker-compose build
+  - docker-compose up -d
+
+after_success:
```

Comment:
This should be moved to `script`. `after_success` does not fail on error.
https://docs.travis-ci.com/user/customizing-the-build/#Breaking-the-Build

Reply:
OK :smile:  09d98784a5749fc488ec33a93ba2482c5b80a9f2